bunsen/bunsen-core-r4/src/main/java/com/cerner/bunsen/definitions/r4/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
nit: run spotless to add header and empty lines at end of file

Reply:
I have not enabled spotless for the Bunsen portion of the repository; I just manually added an empty line (this file is not really needed, I just copied it for consistency).